tools/packaging/scripts/configure-hypervisor.sh

```diff
@@ -319,6 +334,16 @@ generate_qemu_options() {
 	# Don't build the qemu-io, qemu-nbd and qemu-image tools
 	qemu_options+=(size:--disable-tools)
 
+	# Kata Containers may be configured to use the virtiofs daemon.
+	#
+	# But since QEMU 5.2 the daemon is built as part of the tools set
+	# (disabled with --disable-tools) thus it needs to be explicitely
+	# enabled.
+	if gt_eq "${qemu_version}" "5.2.0" ; then
```

Comment:
To clarify; we normally have this disabled in the "configure-hyperivsor" script? Enabling it generally will add a loft of cruft we would otherwise not want?